.rubocop_todo.yml

```diff
@@ -534,6 +534,8 @@ Metrics/AbcSize:
 # ExcludedMethods: refine
 Metrics/BlockLength:
   Max: 432
+  Exclude:
+    - 'lib/tasks/ingest.rake'
```

Comment:
This file already reached maximum lines. So Excluded this file. 
 Can I create a new rake file to this functionality (update all bibs)?

Reply:
Yes, that's cool. Thanks!